File: Python Programs/Area.py
import math

print("Enter 1 to calculate the area of a rectangle")
print("Enter 2 to calculate the area of a square")
print("Enter 3 to calculate the area of a circle")
print("Enter 4 to calculate the area of a triangle")

ch = int(input())

if ch == 1:
    l = float(input("Enter the Length "))
    b = float(input("Enter the Breadth "))
    area = l*b
    print("Area is", area)
elif ch == 2:
    s = float(input("Enter the side "))
    area = s*s
    print("Area is", area)
elif ch == 3:
    r = float(input("Enter the Radius "))
    area = math.pi*r*r
    print("Area is", area)
elif ch == 4:
    b = float(input("Enter the Base "))
    h = float(input("Enter the Height "))
    area = 0.5*b*h
    print("Area is", area)
else:
    print("Wrong Input")

File: MCA Sem 2/C++ codes/area.cpp
#include <iostream>
#include <cmath>

class AREA {
private:
    const double PI = 3.14159265358979323846;

public:
    // Function to calculate area of rectangle
    double calculateArea(double length, double width) {
        return length * width;
    }

    // Function to calculate area of triangle using Heron's formula
    double calculateArea(double a, double b, double c) {
        // Calculate semi-perimeter
        double s = (a + b + c) / 2;
        // Use Heron's formula
        return sqrt(s * (s - a) * (s - b) * (s - c));
    }

    // Function to calculate area of circle
    double calculateArea(double radius) {
        return PI * radius * radius;
    }
};

int main() {
    AREA area;
    double result;

    // Rectangle
    double length, width;
    std::cout << "Enter length of rectangle: ";
    std::cin >> length;
    std::cout << "Enter width of rectangle: ";
    std::cin >> width;
    result = area.calculateArea(length, width);
    std::cout << "Area of rectangle: " << result << std::endl;

    // Triangle
    double a, b, c;
    std::cout << "\nEnter length of first side of triangle: ";
    std::cin >> a;
    std::cout << "Enter length of second side of triangle: ";
    std::cin >> b;
    std::cout << "Enter length of third side of triangle: ";
    std::cin >> c;
    result = area.calculateArea(a, b, c);
    std::cout << "Area of triangle: " << result << std::endl;

    // Circle
    double radius;
    std::cout << "\nEnter radius of circle: ";
    std::cin >> radius;
    result = area.calculateArea(radius);
    std::cout << "Area of circle: " << result << std::endl;

    return 0;
}